src/main/kotlin/dev/rosewood/roseskyblock/manager/WorldManager.kt

```diff
@@ -19,19 +19,16 @@ import org.bukkit.block.Biome
 
 class WorldManager(rosePlugin: RosePlugin) : Manager(rosePlugin) {
 
-    private val _worldGroups = mutableListOf<IslandWorldGroup>()
-    val worldGroups: List<IslandWorldGroup>
-        get() = this._worldGroups.toList()
+    // no need to hide the mutable list, as we discussed
+    val worldGroups = mutableListOf<IslandWorldGroup>()
 
     private var hasReloaded = false
 
     override fun reload() {
-        val worldsFile = File(this.rosePlugin.dataFolder, "worlds.yml")
-        val exists = worldsFile.exists()
+        val worldsFile = File(rosePlugin.dataFolder, "worlds.yml")
 
         val worldsConfig = CommentedFileConfiguration.loadConfiguration(worldsFile)
-        if (!exists)
-            this.saveDefaults(worldsConfig)
+        if (!worldsFile.exists()) saveDefaults(worldsConfig)
```

Comment:
Due to side effects of CommentedFileConfiguration.loadConfiguration() generating a file if it does not exist, this will cause a different result than the original implementation. Revert back to original